### ISSUE TITLE
In version 3.7 for mac, the conversion status and download progress are no longer exposed to accessibility technologies, such as VoiceOver, in the focus order.

### ISSUE THREAD

**Issue by malthej** [openaudible/openaudible#1168]
I'm blind and use a screen reader with OpenAudible, specifically VoiceOver on MacOS. Previously, one could check whether a download or conversion was in progress right beside the table with the book list. However, in the latest version, this information is no longer exposed to VoiceOver. One can locate them with the item chooser, but they are not a part of the focus order as they used to be/should be.
I'm running OpenAudible 3.7 on MacOS Ventura 13.2.1 intel.

**Comment by malthej** [comment]
After further testing, the conversion progress sometimes appears next to "state" in the focus order, but I'm unable to isolate when it appears and when it doesn't.

**Comment by openaudible** [comment]
The conversion and download progress indicators show in the Book Info: State only when a single book is selected (and is in the process of downloading or converting.) You can also look at the Library Status under "Downloading: " and "Converting:" line items-- which will say how many items are actively downloading and are queued to be downloaded. (Same with conversion.) 

I don't see anything in the change log that indicates we did anything for voice over... but haven't experimented with it before.

I just launched Voice Over on my mac. It only seems to be reading the Table and Menus. I wasn't able to figure out how to hover over the library status or audiobook info tables to have it read the contents correctly. Was this previously working and now it isn't?

Happy to try to figure it out with you.

**Comment by openaudible** [comment]
Hey, I was able to turn on voice over for OpenAudible and yes, it appears to work.
Click on the Library Status: Converting/Downloading number. That will select the books currently downloading or converting.
Then select one item from the table. Then hover over the State: in Audiobook Info panel.
Good luck and feel free to reopen this or comment if you have any questions or suggestions.
